lock-api/src/main/java/com/palantir/lock/client/LeasedLockToken.java

```diff
@@ -39,7 +39,7 @@ public final class LeasedLockToken implements LockToken {
     @GuardedBy("this")
     private boolean invalidated = false;
 
-    static LeasedLockToken of(ConjureLockToken serverToken, Lease lease) {
+    public static LeasedLockToken of(ConjureLockToken serverToken, Lease lease) {
```

Comment:
I'm assuming this is just because you want to stick your code in a separate package?

Reply:
yes